core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -188,9 +188,11 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     // Should be replaced with moddable improvements when roads become moddable
     RoadMovementSpeed("Improves movement speed on roads",UniqueTarget.Global),
     RoadsConnectAcrossRivers("Roads connect tiles across rivers", UniqueTarget.Global),
-    RoadMaintenance("[amount]% maintenance on road & railroads", UniqueTarget.Global), 
-    @Deprecated("As of 3.18.17", ReplaceWith("[-amount]% maintenance on road & railroads"))
+    RoadMaintenance("[amount]% maintenance on roads & railroads", UniqueTarget.Global),
```

Comment:
If you're already making a new unique, why not "[amount]% maintenance on [improvementName]"? 
Not only does this enable specificity (e.g. Rome gets -50% maintenance on roads, Germany gets -25% maintenance on Railroads),
That way when we eventually redo the road status improvements to make them not hardcoded (and this will happen eventually) we'll be ready for it

Reply:
This PR also fixes a couple other Social Policies, and I'm still not entirely comfortable digging into the game mechanics, so I'll get rid of the new UniqueType for now to leave room to change it separately.